### ISSUE TITLE
Feature: Merkle Tree

### ISSUE THREAD

**Issue by julio4** [NethermindEth/StarknetByExample#223]
Show an example implementation of Merkle Tree in Cairo

**Comment by raizo07** [comment]
@julio4 I'll like to work on this

**Comment by hudem1** [comment]
Hello @julio4,

Can i jump on this issue ? I would like to give it a try !

**Comment by julio4** [comment]
> @julio4 I'll like to work on this

Let's first finish #214 before starting a new issue for now! I'll assign @hudem1

**Comment by od-hunter** [comment]
Hi @julio4 , Is this open for contribution? If yes can I work it it please?
